src/Model/PhasorDynamics/Bus/BusInfinite.cpp

```diff
@@ -66,10 +66,10 @@ namespace GridKit
      * @param[in] data - structure with bus data
      */
     template <class ScalarT, typename IdxT>
-    BusInfinite<ScalarT, IdxT>::BusInfinite(BusData& data)
-      : BusBase<ScalarT, IdxT>(data.bus_i),
-        Vr_(data.Vm * cos(data.Va)),
-        Vi_(data.Vm * sin(data.Va))
+    BusInfinite<ScalarT, IdxT>::BusInfinite(DataT& data)
+      : BusBase<ScalarT, IdxT>(data.bus_id),
+        Vr_(data.Vr0),
+        Vi_(data.Vi0)
```

Comment:
so we're replacing the usage of `PowerSystemData` with `BusData` but `PowerSystemData` still exists, right? and what was there previously (i'm thinking with regard to the `Vr_(data.Vm * cos(data.Va))` and `Vi_(data.Vm * sin(data.Va))`) is equivalent to what is replacing it (`Vr_(data.Vr0)`, `Vi_(data.Vi0)`). if these are in fact equivalent, then isn't this duplicating input parameters and potentially introducing another place where an error could stem from? maybe this isn't that big of a deal but without an explanation for the change here, it's not clear to me if this was just a placeholder value.

Reply:
Renamed `PowerSystemData` -> `PowerFlowData` in [042ebcd](https://github.com/ORNL/GridKit/pull/116/commits/042ebcdf911e05120837bdb97ee89567f2c2748e). I think it is less confusing now.